specs/derivation.md

```diff
@@ -1074,6 +1074,13 @@ The Ecotone hardfork activation block, contains the following transactions in th
 To not modify or interrupt the system behavior around gas computation, this block will not include any sequenced
 transactions by setting `noTxPool: true`.
 
+In all blocks following the Ecotone hardfork activation, the L1 Attributes Transaction MUST call
+`setL1BlockValuesEcotone`. The `L1Block` contract still contains the `setL1BlockValues` function
+for backwards compatibility but it MUST never be called after the Ecotone hardfork activation.
```

Comment:
"backwards compatibility" is what allows the L2 to be upgrade before the L1 right? 

Is it worth clarifying that?

Reply:
This doesn't really relate to the splitting out L1 and L2 upgrades as different things. This more has to do with L2 genesis generation, we can generate L2 genesis using latest contracts and still be backwards compat with the hardfork not being activated